tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/LanguageServices/Handlers/AbstractEvaluationCommandLineHandlerTests.cs

```diff
@@ -217,6 +217,20 @@ public void AddEvaluationChanges_CanAddItemWithMetadata()
             Assert.Equal("Value", result["Name"]);
         }
 
+        [Fact]
+        public void AddEvaluationChanges_ItemsWithExclusionMetadataAreIgnored()
+        {
+            var handler = CreateInstance(@"C:\Project\Project.csproj");
+
+            var difference = IProjectChangeDiffFactory.WithAddedItems("A.cs;B.cs");
+            var metadata = MetadataFactory.Create("A.cs", ("ExcludeFromCurrentConfiguration", "true"))
+                                          .Add("B,cs", ("ExcludeFromCurrentConfiguration", "false"));
```

Comment:
Is the comma here a typo? If so, is this test working as expected?

```suggestion
                                          .Add("B.cs", ("ExcludeFromCurrentConfiguration", "false"));
```

Reply:
Maybe add a third item `C.cs` with no metadata to explicitly validate behaviour when no metadata is present.